glommio/src/sys/uring.rs

```diff
@@ -1630,37 +1630,29 @@ impl Reactor {
         let mut main_ring = self.main_ring.borrow_mut();
         let mut lat_ring = self.latency_ring.borrow_mut();
 
-        // we need to make sure we consume the latency ring before setting the
-        // preemption timer because consuming this ring may wake up some previously
-        // asleep task queues.
-        consume_rings!(into &mut woke; lat_ring);
```

Comment:
How "tiny" does it have to be for it to matter?
In reality the kernel won't be able to deliver anything with less than 250us reliably anyway